Gruntfile.js

```diff
@@ -17,7 +18,45 @@ module.exports = function(grunt) {
             "build/js/backend/NodeWebkit.js"
         ]
     };
-    
+
+    var buildConfig;
+    try {
+      buildConfig = require('./buildConfig.js');
+      console.log("buildConfig.js present - using it ...");
+    }
+    catch (noBuildConfigFound) {
+      console.log("no buildConfig.js present - using the standard configuration ...");
+      buildConfig = {
+        platforms:
+          [
+              "win32", "osx32", "linux32",
+              "win64", "osx64", "linux64"
+          ],
+        uglifyOptions:{}
+      };
+    }
+
+    console.log("Checking for dependencies ...");
+    try {
+      var bowerFD = fs.openSync("bower_components",'r');
+      fs.closeSync(bowerFD);
+      console.log("... bower is here.");
+    } catch (bowerNotFound) {
+      console.log("... could not find bower_components! Please run `bower install`.");
+      process.exit();
+    }
+
+    try {
+      var nodeFD = fs.openSync("node_modules",'r');
+      fs.closeSync(nodeFD);
+      console.log("... node_modules is here.");
```

Comment:
This check does not seem necessary. I think that Grunt cannot run at all if `node_modules` is not present. It shows the following error:

```
Fatal error: Unable to find local grunt.
```